### PR TITLE
Add a Jira Publisher to publish Dependency Track alerts to Jira tickets

### LINKED ISSUE #2118: Add a Jira Publisher to publish Dependency Track alerts to Jira tickets
Add a Jira Publisher to existing [Publishers](https://docs.dependencytrack.org/integrations/notifications/#configuring-publishers) to publish Dependency Track alerts to Jira tickets

This ER is related to #1583 but narrower in scope in hopes that it will be integrated to the codebase quicker.

### PR DESCRIPTION
**Pull request by Mvld3r**
Closes #2118

### CODE REVIEW

File: docs/_docs/integrations/notifications.md [hunk 1]
```diff
@@ -401,9 +403,11 @@ and publisher to use.
 ![create notification](/images/screenshots/notifications-create.png)
 
 Once the alert is created it can be configured. Start with selecting from the list of available notification groups
-to notify on. Then specify the destination. The destination may be a comma speparated list of email addresses (when the Email publisher is used),
-or a URL. In the case of Slack, Mattermost and Microsoft Teams, this will be the incoming webhook URL generated from each respective
-platform. In the case of the Outbound Webhook publisher, this will be a URL to which to publish the notification.
+to notify on. Then specify the destination:
+- For the Email publisher: it's a comma separated list of email addresses
+- For Slack: Mattermost and Microsoft Teams: it's the incoming webhook URL generated from each respective
```
Comment (nscuro) [inline docs/_docs/integrations/notifications.md:408]: Minor typo:

```suggestion
- For Slack, Mattermost and Microsoft Teams: it's the incoming webhook URL generated from each respective
```

**Review by nscuro — CHANGES_REQUESTED** [review]
Thanks for the PR, @Mvld3r!

Only two minor things to address please.

**Review by nscuro — APPROVED** [review]
Thanks @Mvld3r! 🚀

**Comment by rightphp** [comment]
@nscuro are we planning to release it into 4.7 milestone or earlier to that?

**Pull request merged**